pos_receipt_hide_user/static/src/js/order_receipt.js

```diff
@@ -0,0 +1,50 @@
+odoo.define("pos_receipt_hide_user.OrderReceipt", function (require) {
+  "use strict";
+
+  const OrderReceipt = require("point_of_sale.OrderReceipt");
+  const Registries = require("point_of_sale.Registries");
+
+  const HideUserOrderReceipt = (OriginalOrderReceipt) =>
+    class extends OriginalOrderReceipt {
+      constructor({ hideUserState }) {
+        super(...arguments);
+        this.hideUserState = hideUserState;
+        this.hide_option_from_config = this.env.pos.config.hide_user_option;
+      }
+      get userHidden() {
```

Comment:
```suggestion

      get userHidden() {
```

Reply:
I actually dont understand what you meant here. You meant that there is no need to get useHidden, right?

Reply:
nah nah just some nitpicking: adding a newline